pkg/endpoint/endpoint.go

```diff
@@ -165,11 +165,6 @@ type Endpoint struct {
 	// information of the endpoint has changed
 	identityRevision int
 
-	// LXCMAC is the MAC address of the endpoint
-	//
-	// FIXME: Rename this field to MAC
-	LXCMAC mac.MAC // Container MAC address.
```

Comment:
@tgraf Without looking I guess this will break downgrade as the LXCMAC structure will be filled out with zeros and the connectivity between endpoints will no longer work. I would leave this field and marked as deprecated until Cilium 1.6/1.7 (At the point where we no longer support upgrade from 1.3)

Reply:
Good point

Reply:
I noticed while testing that we need to keep it the address anyway to to proper L3 routing to local endpoints. I've adjusted the commit to only remove the validation.